specification/cognitiveservices/Language.Conversations/main.tsp

```diff
@@ -812,25 +843,254 @@ model ConversationRequestStatistics extends RequestStatistics {
+@doc("Describes the PII categories to exclude for detection. If not provided, 'default' categories will be returned which will vary with the language.")
+enum ConversationPIICategoriesExclude {
+  #suppress "@azure-tools/typespec-azure-core/documentation-required"
+  Address,
+  #suppress "@azure-tools/typespec-azure-core/documentation-required"
+  CreditCardNumber,
+  #suppress "@azure-tools/typespec-azure-core/documentation-required"
+  Email,
+  #suppress "@azure-tools/typespec-azure-core/documentation-required"
+  Person,
+  #suppress "@azure-tools/typespec-azure-core/documentation-required"
+  NumericIdentifier,
+  #suppress "@azure-tools/typespec-azure-core/documentation-required"
+  PhoneNumber,
+  #suppress "@azure-tools/typespec-azure-core/documentation-required"
+  USSocialSecurityNumber,
+}
+
+@added(Versions.v2023_04_15_preview)
+@doc("Enumeration of supported transcript content types.")
+enum TranscriptContentType {
+  #suppress "@azure-tools/typespec-azure-core/documentation-required"
+  lexical,
+  #suppress "@azure-tools/typespec-azure-core/documentation-required"
+  itn,
+  #suppress "@azure-tools/typespec-azure-core/documentation-required"
+  maskedItn,
+  #suppress "@azure-tools/typespec-azure-core/documentation-required"
+  text,
+}
+
+@doc("Task definition for custom conversational summarization.")
+@added(Versions.v2023_04_15_preview)
+model AnalyzeCustomConversationSummarizationTask extends AnalyzeConversationLROTask {
+  @doc("discriminator kind")
+  kind: AnalyzeConversationLROTaskKind.CustomConversationalSummarizationTask;
+
+  @doc("parameters")
+  parameters?: CustomConversationSummarizationTaskParameters;
+}
+
 @doc("Task definition for conversational summarization.")
 @added(Versions.v2023_04_01)
 model AnalyzeConversationSummarizationTask extends AnalyzeConversationLROTask {
   @doc("discriminator kind")
   kind: AnalyzeConversationLROTaskKind.ConversationalSummarizationTask;
 
   @doc("parameters")
-  parameters: ConversationSummarizationTaskParameters;
+  parameters?: ConversationSummarizationTaskParametersBase;
 }
 
 @doc("Supported parameters for a conversational summarization task.")
 @added(Versions.v2023_04_01)
-model ConversationSummarizationTaskParameters extends PreBuiltTaskParameters {
-  ...AbstractiveSummarizationTaskParameterBase;
+model ConversationSummarizationTaskParametersBase {
+  ...AbstractiveSummarizationTaskParametersBase;
 
-  @doc("Supported parameters for a conversational summarization task.")
+  @doc("Array of Summary Aspects")
   summaryAspects: Array<SummaryAspect>;
 }
 
+@doc("Supported parameters for pre-build conversational summarization task.")
+@added(Versions.v2023_04_01)
+model ConversationSummarizationTaskParameters {
+  ...PreBuiltTaskParameters;
+  ...ConversationSummarizationTaskParametersBase;
+}
+
+@doc("Supported parameters for a custom conversation summarization task.")
+@added(Versions.v2023_04_15_preview)
+model CustomConversationSummarizationTaskParameters {
```

Comment:
To maintain a class hierarchy in our SDK, should one of these actually be after `extends` while the other splatted? Splatting both breaks the class hierarchy that no doubt exists already in the SDK which, for autorest, was the first listed for an `allOf` array.